Shared/Extensions/JellyfinAPI/BaseItemDto+Poster.swift

```diff
@@ -74,4 +74,12 @@ extension BaseItemDto: Poster {
             ]
         }
     }
+
+    func cinematicPosterImageSources() -> [ImageSource] {
```

Comment:
This list is incorrect as it has the `.thumb` and `.primary` and breaks for non-episode/series items. Similar to the existing `landscapePosterImageSources(...)`, we should switch on whether it is an episode and return the right backdrop.

```swift
switch type {
case .episode:
    return [seriesImageSource(.backdrop, maxWidth: UIScreen.main.bounds.width)]
default:
    return [imageSource(.backdrop, maxWidth: UIScreen.main.bounds.width)]
}
```